googler

```diff
@@ -836,11 +836,12 @@ def new_connection(domain=None):
     return http.client.HTTPSConnection(domain if domain else server, timeout=45)
 
 
-def quit_program(conn):
-    """Closes a connection and quits the program"""
+def quit_program(exitcode=1):
+    """Close the connection and quits the program."""
 
+    global conn
```

Comment:
`quit_program()` closes the connection passed to it.

`new_connection()` is tied to any global today.

> I think we should only have one open connection at any time.

That is true but we need not use the global `conn` forever. I'm thinking of supporting program options at omniprompt in future. Would add great flexibility.